### PR TITLE
fix tests on windows

### PR DESCRIPTION
**Pull request by kkolman**
i was looking at your project, but the tests were failing on windows - '/' vs '\' stuff

### CODE REVIEW

File: find_test.go [hunk 1]
```diff
@@ -99,7 +102,8 @@ func TestFindWithFileSearchPattern(t *testing.T) {
 func makeAssertPaths(ch chan string) func(f string) bool {
 	var list []string
 	for path := range ch {
-		list = append(list, path)
+		fmt.Println("Found path:", path)
```
Comment (mattn) [inline find_test.go]: remaining debug message

File: find_test.go [hunk 2]
```diff
@@ -99,7 +102,8 @@ func TestFindWithFileSearchPattern(t *testing.T) {
...
+		list = append(list, sanitizePath(path))
```
Comment (mattn) [inline find_test.go]: filepath.ToSlash(path)

File: find_test.go [hunk 3]
```diff
@@ -121,3 +125,8 @@ func defaultOption() Option {
 		},
 	}
 }
+
+// replaces os pathseparator with "/" to make path assertions os-independent
+func sanitizePath(path string) string {
+	return strings.Replace(path, fmt.Sprintf("%c", os.PathSeparator), "/", -1)
+}
```
Comment (mattn) [inline find_test.go]: remove this

**Comment by monochromegane** [comment]
Thanks!

**Pull request merged**